tuf/signed/sign_test.go

```diff
@@ -270,3 +248,59 @@ func TestSignWithX509(t *testing.T) {
 	assert.Len(t, testData.Signatures, 1)
 	assert.Equal(t, tufRSAx509Key.ID(), testData.Signatures[0].KeyID)
 }
+
+func TestSignWithOldKey(t *testing.T) {
```

Comment:
Nit:  the test name implies to me that this is testing that we can sign with old keys - could we rename this to `TestSignRemovesValidSignaturesByOldKeys`?